common/autoend

```diff
@@ -0,0 +1,428 @@
+#!/usr/bin/wish
+# IBM_PROLOG_BEGIN_TAG 
+# This is an automatically generated prolog. 
+#  
+# esw_dev_tools src/aipl/x86/autoend 1.11 
+#  
+# Licensed Materials - Property of IBM 
+#  
+# Restricted Materials of IBM 
+#  
+# COPYRIGHT International Business Machines Corp. 2009,2012 
+# All Rights Reserved 
+#  
+# US Government Users Restricted Rights - Use, duplication or 
+# disclosure restricted by GSA ADP Schedule Contract with IBM Corp. 
+#  
+# IBM_PROLOG_END_TAG 
+
+regsub {.*/} $argv0 {} program_name
+
+# Source include files (from the same directory as this file).
+source [file dirname [info script]]/gen_tcl_print_funcs 
```

Comment:
Why is this in the common dir?  Doesn't seem like something other people will use?  Specific to boot testing right?